src/formatDate.js

```diff
@@ -50,7 +50,40 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = toFormat[3];
+  const dataArray = date.split(fromFormat[3]);
+  let toDate = '';
+
+  if (fromFormat.includes('YYYY') && toFormat.includes('YY')) {
+    const index = fromFormat.indexOf('YYYY');
+
+    fromFormat[index] = 'YY';
```

Comment:
```suggestion
    // fromFormat[index] = 'YY'; <-- This is a bad practice!
```
Here you've modified function argument. It's forbidden. Also it's an array - so your change affected original object too.

Reply:
corrected